src/pymovements/datasets/colagaze.py

```diff
@@ -0,0 +1,185 @@
+        'precomputed_reading_measures'.
+
+    resources: dict[str, list[dict[str, str]]]
+        A list of dataset gaze_resources. Each list entry must be a dictionary with the following
+        keys:
+        - `resource`: The url suffix of the resource. This will be concatenated with the mirror.
+        - `filename`: The filename under which the file is saved as.
+        - `md5`: The MD5 checksum of the respective file.
+
+    extract: dict[str, bool]
+        Decide whether to extract the data.
+
+    experiment: Experiment
+        The experiment definition.
+
+    filename_format: dict[str, str]
+        Regular expression which will be matched before trying to load the file. Namedgroups will
+        appear in the `fileinfo` dataframe.
+
+    filename_format_schema_overrides: dict[str, dict[str, type]]
+        If named groups are present in the `filename_format`, this makes it possible to cast
+        specific named groups to a particular datatype.
+
+    custom_read_kwargs: dict[str, dict[str, Any]]
+        If specified, these keyword arguments will be passed to the file reading function.
+
+    Examples
+    --------
+    Initialize your :py:class:`~pymovements.dataset.Dataset` object with the
+    :py:class:`~pymovements.datasets.CoLAGaze` definition:
+
+    >>> import pymovements as pm
+    >>>
+    >>> dataset = pm.Dataset("CoLAGaze", path='data/CoLAGaze')
+
+    Download the dataset resources:
+
+    >>> dataset.download()# doctest: +SKIP
+
+    Load the data into memory:
+
+    >>> dataset.load()# doctest: +SKIP
+    """
+
+    # pylint: disable=similarities
+    # The PublicDatasetDefinition child classes potentially share code chunks for definitions.
+
+    name: str = 'CoLAGaze'
+
+    long_name: str = 'Corpus of Eye Movements for Linguistic Acceptability'
+
+    has_files: dict[str, bool] = field(
+        default_factory=lambda: {
+            # Note: Gaze is currently disabled due to metadata mismatch issue.
+            'gaze': True,
+            'precomputed_events': True,
+            'precomputed_reading_measures': True,
+        },
+    )
+
+    resources: dict[str, list[dict[str, str]]] = field(
+        default_factory=lambda: {
+            'gaze': [
+                {
+                    'resource':
+                    'https://files.au-1.osf.io/v1/resources/gj2uk/providers/osfstorage/'
+                    '67e14ce0f392601163f33215/?view_only=a8ac6e0091e64d0a81d5b1fdec9bab6e&zip=',
+                    'filename': 'raw_data.zip',
+                    'md5': None,  # type: ignore
+                },
+            ],
+            'precomputed_events': [
+                {
+                    'resource':
+                    'https://files.au-1.osf.io/v1/resources/gj2uk/providers/osfstorage/'
+                    '67e14ce0f392601163f33215/?view_only=a8ac6e0091e64d0a81d5b1fdec9bab6e&zip=',
+                    'filename': 'fixations.zip',
+                    'md5': None,  # type: ignore
```

Comment:
why is the checksum null?

Reply:
checksum changes with every download

Reply:
I see. I created issue #1152 to resolve this.